src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -97,6 +100,15 @@ public Command parseCommand(String userInput) throws ParseException {
             HistoryList.addToHistory(userInput);
             return new SortCommandParser().parse(arguments);
 
+        case AddToGroupCommand.COMMAND_WORD:
+            return new AddToGroupCommandParser().parse(arguments);
+
+        case UngroupCommand.COMMAND_WORD:
+            return new UngroupCommandParser().parse(arguments);
+
+        case GroupCommand.COMMAND_WORD:
+            return new GroupCommandParser().parse(arguments);
+
```

Comment:
There seems to be an error in the AddreddBookParserTest: parseCommand that has caused the current implemented Tests to fail. Maybe there is an unexpected behaviour caused by the addition of the Group commands.